customs.js

```diff
@@ -55,6 +57,13 @@
       }
     )
     .then(
+      (function (result) {
+        if (result.lockout) {
+          return this.db.lockAccount(account)
```

Comment:
In yesterday's fxa-web-coordination meeting, @ckarlof mentioned we may want to merge this, but only lock out accounts that meet a certain criteria. Perhaps only actually lock the account if the account is a mozilla.com or restmail.com address.

Ideas tossed around:
* use an able-like standin that provides feature-toggles.
* use a regular expression and check the email address.

I'm partial to the regular expression, for its simplicity. We are going to have to update the fxa-auth-server when the real able is ready, already. Instead of removing the standin able and replacing it with the real able, we could just remove the regexp when we are ready to enable the functionality for everyone.

@ckarlof, @rfk, @chilts and @dannycoates - care to weigh in?